tests/contracts.hpp.in

```diff
@@ -14,16 +14,20 @@ struct contracts {
    static std::vector<char>    wrap_abi() { return read_abi("${CMAKE_BINARY_DIR}/../contracts/eosio.wrap/eosio.wrap.abi"); }
    static std::vector<uint8_t> bios_wasm() { return read_wasm("${CMAKE_BINARY_DIR}/../contracts/eosio.bios/eosio.bios.wasm"); }
    static std::vector<char>    bios_abi() { return read_abi("${CMAKE_BINARY_DIR}/../contracts/eosio.bios/eosio.bios.abi"); }
+   static std::vector<uint8_t> sendinline_wasm() {return read_wasm("${CMAKE_BINARY_DIR}/../contracts/test_contracts/sendinline/sendinline.wasm"); }
```

Comment:
I don't know why we called the struct `contracts::util` (it probably should have been `contracts::test`), but it seems we put all contracts that are part of the actual eosio.contracts product in `contracts` and all the remaining contracts needed for testing into `contracts::util`. So to continue that convention, please move these two functions for the `sendinline` contract to `util`.

Reply:
Moved to `util` namespace.